.babelrc

```diff
@@ -1,9 +1,17 @@
 {
-  "presets": ["es2015"],
-  "plugins": [
-    "transform-flow-strip-types",
-    "transform-es3-property-literals",
-    "transform-es3-member-expression-literals",
-    ["transform-es2015-for-of", {"loose": true}]
-  ]
+  "plugins": ["transform-flow-strip-types"],
+  "env": {
+    "production":{
+      "presets": [["es2015", {"modules": false}]]
+    },
+    "test": {
+      "presets": ["es2015"],
```

Comment:
I think it actually ignores transpiling the imports and expots because it can't be loaded in browser when modules is false.  Errors with import keyword